examples/error-destructure-binding-pattern/error_destructure_binding_pattern.bal

```diff
@@ -1,36 +1,40 @@
 import ballerina/io;
 
-type SampleError error<string, map<anydata|error>>;
+type SampleErrorData record {
+    string info;
```

Comment:
SampleErrorData must contain the following two fields. 
```
string message?;
error cause?;
```

Reply:
This is working due to a bug. https://github.com/ballerina-platform/ballerina-lang/issues/17121

Reply:
Added the fields as suggested.